jenkins/scripts/select-compiler.sh

```diff
@@ -20,7 +20,12 @@ if [ "$SELECT_ARCH" = "PPC64LE" ]; then
 
   echo "Setting compiler for Node version $NODEJS_MAJOR_VERSION on ppc64le"
 
-  if [ "$NODEJS_MAJOR_VERSION" -gt "9" ]; then
+  if [ "$NODEJS_MAJOR_VERSION" -gt "11" ]; then
```

Comment:
We've been building with binutils-2.26 for 9+ so I think we should continue to do that, we might adjust for 11+ but I'm not sure it's worth it.

We should remove the Why? comment through or have an issue at least to take an action if we believe we need to.  My understanding from Rod is that it was needed to ensure we get OpenSSL assembly acceleration.

Reply:
replaced comment with a ref to information above